source/websocket_bootstrap.c

```diff
@@ -160,10 +165,16 @@ int aws_websocket_client_connect(const struct aws_websocket_client_connection_op
     ws_bootstrap->websocket_frame_payload_callback = options->on_incoming_frame_payload;
     ws_bootstrap->websocket_frame_complete_callback = options->on_incoming_frame_complete;
     ws_bootstrap->handshake_request = aws_http_message_acquire(options->handshake_request);
+    aws_byte_buf_init(&ws_bootstrap->expected_sec_websocket_accept, ws_bootstrap->alloc, 0);
```

Comment:
super trivial: Can we init the buffer within `s_ws_bootstrap_calculate_sec_websocket_accept`? Just that `aws_byte_buf_init` and `aws_byte_buf_reserve` to one single `aws_byte_buf_init`? Unless we plan to use `s_ws_bootstrap_calculate_sec_websocket_accept` somewhere else that already have a buffer initialized.

Reply:
sure